apps/projects/app/components/Content/Issues.js

```diff
@@ -99,7 +101,7 @@ class Issues extends React.PureComponent {
 
   handleFiltering = filters => {
     // TODO: why is reload necessary?
```

Comment:
I didn't spend too long trying to get this to work without the `reload` state variable, but this is generally caused by the fact that state was stored in two different locations, and state updates on the parent not being propogated to the child that actually needs to get rerendered on a state change (due to a changed filter)